website/src/pages/api-documentation/index.astro

```diff
@@ -24,15 +24,15 @@ const BUTTON_CLASS =
 
         <div>
             <p class='mt-4 mb-4'>
-                We offer a
+                There is a
                 <a
                     href='https://swagger.io/tools/swagger-ui/'
                     class='underline text-primary-900 hover:text-primary-800 hover:no-underline'
                 >
                     Swagger UI
                 </a>
-                for documentation and direct interaction with our APIs. For more tips on how to use our API we recommend
-                starting out with our
+                for documentation and direct interaction with the APIs. For more tips on how to use the API, it is recommended
+                to start with the
```

Comment:
I rephrased this as it seems a bit strange to use "we" if the instance is hosted by someone outside of the Loculus/Pathoplexus team.